src/jquery.valueview.valueview.js

```diff
@@ -183,7 +183,7 @@ $.widget( 'valueview.valueview', PARENT, {
 		value: null,
 		language: null,
 		autoStartEditing: false,
-		parseDelay: 300,
+		parseDelay: 500,
```

Comment:
i don't think it's necessary and if you think it's best not to change, then ok :)